services/core-web/common/utils/Validate.ts

```diff
@@ -237,13 +238,19 @@ export const validateIncidentDate = memoize((reportedDate) => (value) =>
     : undefined
 );
 
-// eslint-disable-next-line consistent-return
-export const validateSelectOptions = memoize((data, allowEmptyData = false) => (value) => {
+/**
+ * @param data: options to choose from
+ * @param allowEmptyData: perform validation with empty data array (free text on empty data set will be invalid)
+ */
+export const validateSelectOptions = memoize((data: IOption[], allowEmptyData = false) => (value):
+  | string
+  | undefined => {
   if (value && (data?.length > 0 || allowEmptyData)) {
     return data?.find((opt) => opt.value === value) !== undefined
       ? undefined
       : "Invalid. Select an option provided in the dropdown.";
```

Comment:
I realize this is pre-existing, but this text bothers me.  "Invalid." Doesn't seem like the smoothest UI choice.  Even something like "Selection Invalid." would be better I think.